### PR TITLE
Bring `erc20` and `nft` on par in V2

### PR DESCRIPTION
**Pull request by chuxinh**
Add additional `erc20` and `nft` tokens that exist in V1 to V2

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/tokens/optimism/tokens_optimism_erc20.sql [hunk 1]
```diff
@@ -181,12 +181,34 @@ SELECT LOWER(contract_address) AS contract_address, symbol, decimals
...
+,('0x922d641a426dcffaef11680e5358f34d97d112e1','anyFXS',18)
+,('0xa3a538ea5d5838dc32dde15946ccd74bdd5652ff','sINR', 18)
+,('0xee9801669c6138e84bd50deb500827b776777d28','O3', 18)
+,('0xb12c13e66ade1f72f71834f2fc5082db8c091358','ROOBEE', 18)
+,('0xc22885e06cd8507c5c74a948c59af853aed1ea5c','USDD', 18)
+,('0xd6909e9e702024eb93312b989ee46794c0fb1c9d','BICO', 18)
+,('0x62bb4fc73094c83b5e952c2180b23fa7054954c4','PTaOptUSDC', 6)
+,('0x66e8617d1df7ab523a316a6c01d16aa5bed93681','SHACK', 18)
+,('0x00a35fd824c717879bf370e70ac6868b95870dfb','IB', 18)
+,('0xf899e3909b4492859d44260e1de41a9e663e70f5','RADIO', 18)
```
Comment (soispoke) [inline models/tokens/optimism/tokens_optimism_erc20.sql:193]: Please remove one of these two lines to avoid duplicates @chuxinh

**Review by soispoke — CHANGES_REQUESTED** [review]
lgtm, thansk @chuxinh for this submission!

File: models/tokens/optimism/tokens_optimism_erc20.sql (hunk 1 above)
Comment (chuxinh) [reply models/tokens/optimism/tokens_optimism_erc20.sql:193]: done, thanks!

Comment (soispoke) [reply models/tokens/optimism/tokens_optimism_erc20.sql:193]: thanks @chuxinh, lgtm !

**Review by soispoke — APPROVED** [review]

**Pull request merged**